DO logs in dash (#24994)
cloudflare/cloudflare-docs: src/content/changelog/workers/2025-09-08-logs-for-durable-objects.mdx

---
title: You can now view logs directly in the Durable Objects dashboard 

> description: View, search and filter logs within the Durable Objects dashboard

Reviewer: Add `products` tag for DOs and Workers

date: 2025-09-08
---
import { WranglerConfig } from "~/components";

[Durable Objects](https://developers.cloudflare.com/durable-objects/) now have a dedicated logging dashboard that makes it easier to debug and monitor your applications
![Logs in Durable Objects dashboard](~/assets/images/workers-observability/2025-09-08-logs-for-durable-objects.png)

**With these updates, you can:**
* View logs by clicking into a Durable Object(DO) namespace within the [DO dashboard](https://dash.cloudflare.com/?to=/:account/workers/durable-objects) 
* Add filters and search through all DO related log events
* Identify the Durable Object instance ID (displayed as `$workers.durableObjectId`) to understand which specific instance generated the log entry. 
![instance ID in DO logs](~/assets/images/workers-observability/2025-09-08-durableobject-id.png)

**To get started:**\

> Set the following on the Worker managing your Durable Object namespace:

Reviewer: ```suggestion
Set the following on the Worker that defines your Durable Object class:
```

---
title: You can now view logs directly in the Durable Objects dashboard 
description: View, search and filter logs within the Durable Objects dashboard
date: 2025-09-08
products:
  - workers
  - durable-objects
---
import { WranglerConfig } from "~/components";

![Logs in Durable Objects dashboard](~/assets/images/workers-observability/2025-09-08-logs-for-durable-objects.png)

[Durable Objects](https://developers.cloudflare.com/durable-objects/) now have a dedicated logging dashboard that makes it easier to debug and monitor your applications.

> **With these updates, you can:**

Reviewer 2: ```suggestion
With these updates, you can:
```

> * View logs by clicking into a Durable Object(DO) namespace within the [DO dashboard](https://dash.cloudflare.com/?to=/:account/workers/durable-objects) 

Reviewer 2: ```suggestion
* View logs by clicking into a Durable Object (DO) namespace within the [DO dashboard](https://dash.cloudflare.com/?to=/:account/workers/durable-objects) 
```

* Add filters and search through all DO related log events

> * Identify the Durable Object instance ID (displayed as `$workers.durableObjectId`) to understand which specific instance generated the log entry. 

Reviewer 2: ```suggestion
* Identify the Durable Object instance ID (displayed as `$workers.durableObjectId`) to understand which specific instance generated the log entry
```
